nautobot/core/views/mixins.py

```diff
@@ -985,7 +982,7 @@ def _process_bulk_update_form(self, form):
             if field not in form_custom_fields + form_relationships + ["pk"] + ["object_note"]
         ]
         nullified_fields = request.POST.getlist("_nullify")
-        form_cf_to_key = {f"cf_{cf.slug}": cf.name for cf in CustomField.objects.get_for_model(model)}
+        form_cf_to_key = {f"cf_{cf.key}": cf.name for cf in CustomField.objects.get_for_model(model)}
```

Comment:
Why did we decide to stick with the `form_to_cf_key` pattern here, but not the other place? IMO we should align them, and make the appropriate changes below on lines 1013 and 1015 to use slicing.